vllm/worker/xpu_worker.py

```diff
@@ -174,5 +175,7 @@ def init_worker_distributed_environment(self) -> None:
         ensure_model_parallel_initialized(
             parallel_config.tensor_parallel_size,
             parallel_config.pipeline_parallel_size)
-        # global all_reduce needed for overall oneccl warm up
-        torch.distributed.all_reduce(torch.zeros(1).xpu())
```

Comment:
What about keeping this line? I remembered your current change is the previous logic we used before and we changed it based on last enablement of TP+PP in the cluster. Seems now the communication behavior changes again. So maybe keep this line to cover more cases.